xml/System.Net.Security/SslStream.xml

```diff
@@ -4318,6 +4318,7 @@ This property gets the cipher suite that is going to be used in the communicatio
         <value>An X509Certificate object that represents the certificate supplied for authentication or <see langword="null" /> if no certificate was supplied.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[  
+If the property is accessed, the remote certificate will not be disposed when the SslStream instance is disposed. The caller of the property is responsible for disposing the returned X509Certificate object.
```

Comment:
Should the reference to X509Certificate be either a reference or at least formatted as code?